drift-transport-netty/src/main/java/com/facebook/drift/transport/netty/client/ConnectionFactory.java

```diff
@@ -61,10 +63,15 @@
     @Override
     public Future<Channel> getConnection(ConnectionParameters connectionParameters, HostAndPort address)
     {
+        Class socketChannelClass = NioSocketChannel.class;
+        if (connectionFactoryConfig.isNativeTransportEnabled() && Epoll.isAvailable()) {
```

Comment:
I'm not sure if it makes much sense to silently fallback to `Nio` if native transport is explicitly enabled but not available. 

How about 

```
if (connectionFactoryConfig.isNativeTransportEnabled()){
  checkState(Epoll.isAvailable(), "native transport is not available);
  socketChannelClass = EpollSocketChannel.class;
}
```

Reply:
done